### PR TITLE
Improvements to "masteryScore" and "moveOn" in LaunchData section

### PR DESCRIPTION
**Pull request by brianjmiller**
Addresses #318. Tried to be as consistent as possible with the language already existing in "launchParameters".

### CODE REVIEW

File: cmi5_spec.md [hunk 1]
```diff
@@ -1035,10 +1035,10 @@ The properties for the "LMS.LaunchData" document are described below.
 <table>
   <tr><th colspan=2 align="left">masteryScore</th></tr>
   <tr><th align="right" nowrap>Description:</th><td>The "masteryScore" from the cmi5 Course Structure.</td></tr>
-  <tr><th align="right" nowrap>LMS Required:</th><td>Yes</td></tr>
-  <tr><th align="right" nowrap>AU Required:</th><td>Yes</td></tr>
-  <tr><th align="right" nowrap>LMS Usage:</th><td>The LMS MUST include the "masteryScore" value based on the value defined in the course structure for the AU being launched.</td></tr>
-  <tr><th align="right" nowrap>AU Usage:</th><td>The AU MUST issue Passed or Failed statements based on the "masteryScore" provided. (See Sections 9.3.6 and 9.3.7)</td></tr>
+  <tr><th align="right" nowrap>LMS Required:</th><td>If the "masteryScore" was defined by the course designer in the Course Structure, the LMS MUST include the "masteryScore" in the State API document.</td></tr>
```
Comment (cawerkenthin) [inline cmi5_spec.md]: The LMS Administrator may override the masteryScore, so the referencing the value as coming solely from the course designer is not correct.

Comment (brianjmiller) [inline cmi5_spec.md]: That seems like a requirements change, can the LMS decide a masteryScore different than the course structure? (Based on today's call it seems some feel yes, but I think that'd have to be discussed.)

Comment (brianjmiller) [inline cmi5_spec.md]: Note that if this changes the verb sections need to be revisited as they talk about comparing the score as in the AU with the "masteryScore" from the "Course Structure" not from the "LMS Launch Data" (which perhaps ought to be an issue anyways.)

Comment (cawerkenthin) [inline cmi5_spec.md]: Yes, the LMS Administrator can override the masteryScore.  This is why the masterScore is added to the State.

Comment (brianjmiller) [inline cmi5_spec.md]: I don't think that the LMS is allowed to change the value is captured anywhere in the spec currently. As far as I've seen it only indicates it is passed through, perhaps open an issue? As I noted if that is the desired behavior minimally the language in the verb section is problematic.

**Pull request merged**